examples/sw/simple_system/common/link.ld

```diff
@@ -19,6 +19,16 @@ _stack_start   = ORIGIN(stack) + LENGTH(stack);
 _entry_point = _vectors_start + 0x80;
 ENTRY(_entry_point)
 
+/* The tohost address is used by Spike for a magic "stop me now" message. This
+   is set to equal SIM_CTRL_CTRL (see simple_system_regs.h), which has that
+   effect in simple_system simulations. Note that it must be 8-byte aligned.
+
+   We don't read data back from Spike, so fromhost is set to some dummy value:
+   we place it just above the top of the stack.
+ */
+tohost   = 0x20008;
+fromhost = _stack_start + 0x10;
```

Comment:
So Spike may write something to `fromhost`?

Reply:
I think that Spike supports some form of host <-> target protocol for debugging using OpenOCD. I guess that this is how they send information in the other direction.

Reply:
I think that Spike supports some form of host <-> target protocol for debugging using OpenOCD. I guess that this is how they send information in the other direction.